Tools/python/Samples.py

```diff
@@ -72,7 +77,12 @@ def _set_signal_stuff(sample):
     sample.tau = _tau(sample)
     sample.mass = _mass(sample)
     sample.latex = _latex(sample)
-    sample.xsec = 1e-3
+    #sample.xsec = 1e-3
+    if (sample.name.startswith('WplusH')):
+        sample.xsec = 3*(9.426e-02)*0.01# xsec(pp->3*(W+->lv)H) * br(1% of H->SS) for 3 lepton flavours as in https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV#ppWH_Total_Cross_Section_with_ap and https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/production/2017/13TeV/Higgs/WplusHJ_HanythingJ_NNPDF31_13TeV/HWplusJ_HanythingJ_NNPDF31_13TeV_M125_Vleptonic.input  
```

Comment:
We need the same for ZH, right? Or we will be surprised when we compare the two!! Alternatively, we could set it to 1fb as we do for the other samples and scale accordingly after the fact (for both WH and ZH).